docker-bake.hcl

```
@@ -35,10 +38,15 @@ target "docker-metadata-action" {
 }
 
 group "default" {
-  targets = ["image"]
+  targets = ["image-local"]
+}
+
+target "image-local" {
+  inherits = ["_common"]
+  tags = ["${IMAGE_LOCAL}"]
```

Would be better to provide an output otherwise it warns there is none:

```suggestion
target "image-local" {
  inherits = ["_common"]
  tags = ["${IMAGE_LOCAL}"]
  output = ["type=docker"]
```

---

Woops, this actually needs to be `type=image` - we don't want it in the local image store, we want it pushed to the local registry.